BedrockServer.cpp

```diff
@@ -1576,8 +1576,10 @@ void BedrockServer::_reply(unique_ptr<BedrockCommand>& command) {
     // Finalize timing info even for commands we won't respond to (this makes this data available in logs).
     command->finalizeTimingInfo();
 
-    // Don't reply to commands with pseudo-clients (i.e., commands that we generated by other commands).
+    // Don't reply to commands with pseudo-clients (i.e., commands that we generated by other commands, or using
+    // `Connection: forget`.
     if (command->initiatingClientID < 0) {
+        command->handleFailedReply();
```

Comment:
We set initiatingClientID to `-1` for `Connection: forget` here: https://github.com/Expensify/Bedrock/blob/434fce36e68179f5476f3c21c772d1dd510e2613/BedrockServer.cpp#L2228

So we should call `handleFailedReply` here for that case.